set.mm

```diff
@@ -641864,25 +641864,27 @@ D Fn ( I ... ( M - 1 ) ) /\
   ${
     ifexd.1 $e |- ( ph -> A e. V ) $.
     ifexd.2 $e |- ( ph -> B e. W ) $.
-    $( Conditional operator existence.  (Contributed by SN, 26-Jul-2024.) $)
+    $( Existence of the conditional operator (deduction form).  EDITORIAL: If
+       this theorem is moved to main, use it to shorten ~ ifex .  (Contributed
+       by SN, 26-Jul-2024.) $)
```

Comment:
It would be nice if in a later MR, you can move ifexd to Main and see which order of proving ifcl(d) and ifex(d|g|) makes the overall proof shorter.
Actually, it looks like there is a theorem missing, which would be
```
|- ( ( A e. U /\ B e. V ) -> if ( ph , A , B ) e. ( U u. V ) )
```
and its deduction form too.

Reply:
@icecream17 : I'll do this unless you tell me you are working on it.  Are you ?

Reply:
ifexd moved to Main in #4139. I took a note to add `|- ( ( A e. U /\ B e. V ) -> if ( ph , A , B ) e. ( U u. V ) )`.

Resolving.